components/generator/group.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div v-show="isGroup && links.length > 0">
+    <div class="group animated fadeIn">
+      <link-display
+        v-for="(link, index) in links"
+        :key="index"
+        :link="link" />
+    </div>
+
+    <button
+      @click.once="generateRoute"
+      class="cursor-pointer with-hover">
+      <c-icon icon="layer-group" class="mr" />
+      <span v-text="$t('submitText')" />
+    </button>
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters, mapActions } from 'vuex'
+import storageService from '@/services/storage-service.js'
+import linkDisplay from '@/components/generator/link.vue'
+
+export default {
+  components: {
+    linkDisplay
+  },
+  computed: {
+    ...mapState('generator', [
+      'links',
+      'shortcut',
+      'isGroup'
+    ]),
+    ...mapGetters('generator', [
+      'qualifiedShortcut'
+    ])
+  },
+  methods: {
+    ...mapActions('generator', [
+      'generate'
+    ]),
+    ...mapActions('snackbar', [
+      'showSnackbar'
+    ]),
+
+    generateRoute () {
+      this.generate().then(() => {
```

Comment:
UX here needs to be improved - while waiting the page is static, needs something to let the user know what is happening

Reply:
Sorted